extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -44,6 +44,32 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
   let impl;
   let element;
 
+  /**
+   * Creates an iframe promise, and instantiates element and impl, adding the
+   * former to the document of the iframe.
+   * @param {{width, height, type}} config
+   * @return The iframe promise.
+   */
+  function createImplTag(config) {
```

Comment:
Outside of this PR, let's discuss if/how we could save the need to duplicate tests across adsense/double imples